File: Python/Programmars/Level_1/수박수박수박수박수박수?.py
def solution(n):
    answer = ''
    for i in range(1, n+1):
        if i % 2 != 0:
            answer += '수'
        else:
            answer += '박'
    return answer


print(solution(3))  # "수박수"
print(solution(4))  # "수박수박"

File: JavaScript/Programmars/Level_1/수박수박수박수박수박수?.js
// 내 풀이
function solution(n) {
  let answer = '';
  for (let i = 0; i < n; i++) {
    answer += i % 2 === 0 ? '수' : '박';
  }
  return answer;
}

console.log(solution(3)); // '수박수'
console.log(solution(4)); // '수박수박'

// 다른 풀이
function solution(n) {
  return '수박'.repeat(n).substring(0, n);
}
